### DISCUSSION TITLE
Confusing documentation for `mx.sym.MakeLoss`

### DISCUSSION THREAD

**Discussion by fhieber** [apache/incubator-mxnet#14088] [Q&A]
The [documentation of `mx.sym.MakeLoss`](http://mxnet.incubator.apache.org/api/python/symbol/symbol.html?highlight=makeloss#mxnet.symbol.MakeLoss) is highly confusing. To my understanding the only thing `MakeLoss` does is that it wraps an existing symbol and defines that it does not require a head gradient when used in optimization. Further, it seems that the output of the `forward()` call to a MakeLoss-symbol is the output of the `forward()` call of the wrapped symbol. That is, MakeLoss just passes through its input data in `forward()`.

However, the documentation states the following:
"The output of this function is the gradient of loss with respect to the input data."

What does this mean? I read it as if the output of `forward()` is the same as the output of `backward()`, namely the gradient of the symbol, MakeLoss wraps. But this does not seem to be true.

**Comment by andrewfayres** [comment]
@mxnet-label-bot add [documentation, question]

**Comment by andrewfayres** [comment]
@mxnet-label-bot add [doc]
